pc/src/usb.rs

```diff
@@ -35,9 +35,9 @@ pub(crate) struct Players {
 	/// The connection establishment is already finish
 	/// and message can be send/receive to/from player,
 	/// if entry is `Some`.
-	players: [Option<Player>; 4],
+	pub(crate) players: [Option<Player>; 4],
 	///uart devices, wich where not classificated as pybadge yet
-	possible_players: Vec<Player>
+	pub(crate) possible_players: Vec<Player>
```

Comment:
`possible_players` player should keep private, because it is only relevant for the connection establishment and does not include any data which are relevant for other modules.

Making `players` pub is fine.